src/test/hardhatRPC.ts

```diff
@@ -4,7 +4,8 @@ import { ethers } from "ethers";
 
 const RPC_URL = String(process.env.RPC_URL);
 
-const latestSnapshot: string[] = [];
+// eslint-disable-next-line prefer-const
+let latestSnapshot: string[] = [];
```

Comment:
I think the const is fine since we're not reassigning to latestSnapshot, we're just messing with its insides?

Reply:
That would mean this is a global const, which would mean it should be named `LATEST_SNAPSHOT` instead. Should we rename it? Or just change it to a const and disable the naming convention instead?

Reply:
That's an interesting question.  What does "constancy" mean really?  To me the all caps has an implied meaning of not being permuted ever, not even on the inside.  However a `let` implies and allows reassignment entirely.  This is also in test so, maybe a let is fine?